transport.go

```diff
@@ -2,9 +2,13 @@ package libp2pquic
 
 import (
 	"context"
+	"crypto/sha256"
```

Comment:
We usually use https://github.com/minio/sha256-simd for performance.

Reply:
I don't think performance is critical here, since the only time we calculate a hash is when `NewTransport` is called. I'm ok with changing it to keep things consistent though.

This will be obsoleted with https://github.com/libp2p/go-libp2p-core/pull/131 anyway.